src/options.ml

```diff
@@ -1,4 +1,8 @@
-let default_allow_unused_code_warnings = false
+module Allow_unused_code_warnings = struct
```

Comment:
What do you think about using a more generic name here, such as `Force_flag`?

Reply:
I like it, thanks! Done.